src/stateProviders/provideSavedQueries.js

```diff
@@ -15,28 +15,39 @@ export default provideState({
     removeDeletingId: update((state, id) => ({
       deletingIds: state.deletingIds.filter(dId => dId !== id),
     })),
-    getQueries: (effects, egoId) =>
-      effects
+    getQueries: (effects, egoId) => {
+      const jwt = localStorage.getItem('EGO_JWT');
```

Comment:
same comment here (and line 35 below), should provide jwt via state provider rather than directly from localstorage